pkg/roachprod/roachprod.go

```diff
@@ -840,6 +831,8 @@ func getLabels(v vm.VM) map[string]string {
 		"host_ip":  v.PrivateIP,
 		"project":  v.Project,
 		"zone":     v.Zone,
+		"tenant":   install.SystemInterfaceName,
```

Comment:
Tiny nit(s): `getLabels` to me seems like it would get the labels from an already created config, not construct them. Maybe something like `getDefaultLabels` or `createLabels` would be a slightly more descriptive name? 

Also maybe we should start being explicit about VM labels vs Prom labels, since we also have `AddLabels` and `RemoveLabels` in this file.

Reply:
I will change it to createLabels